packages/cbioportal-utils/src/mutation/MutationAnnotator.ts

```diff
@@ -290,17 +292,15 @@ export function annotateMutation(
 export function initAnnotatedMutation(
     mutation: Partial<Mutation>
 ): Partial<Mutation> {
-    const deepClone = toJS(mutation);
```

Comment:
`toJS` is now significantly slower. we don't really need to deep clone here, just a shallow copy should be good enough.